services/system/Accounts/plugin/src/errors.rs

```diff
@@ -26,6 +27,11 @@ impl ErrorType {
                 producer: my_plugin_id(),
                 message: format!("Invalid account number: {}", msg),
             },
+            ErrorType::QueryError => Error {
+                code: self as u32,
+                producer: my_plugin_id(),
+                message: format!("Failed to post graphql query: {}", msg),
```

Comment:
This QueryError says, "failed to post..." but it should probably just say something more general, like, "GraphQL query error: ..." since it's not only emitted when posting fails.